priv/posts/security/20241217151548_epmd-public-exposure.md

```diff
@@ -44,6 +44,8 @@ If left unsecured, exposed Erlang Distribution ports let attackers gain a footho
   ```
   Ensuring that distribution ports are only accessible via localhost or a private network interface further reduces the attack surface.
 
+- **Enable epmd-less mode in your releases**: Both Erlang and Elixir tooling allows you to use the Erlang Distribution without exposing EPMD. Rebar3 enables this option by default. In Elixir, [it is part of your generated release configuration](https://hexdocs.pm/mix/Mix.Tasks.Release.html#module-epmd-less-deployment).
```

Comment:
```suggestion
- **Enable epmd-less mode in your releases**: Both Erlang and Elixir tooling allows you to use the Erlang Distribution without EPMD. Rebar3 releases run with a port number set through the environment variable `ERL_DIST_PORT` at runtime will enable this automatically. In Elixir, [it is part of your generated release configuration](https://hexdocs.pm/mix/Mix.Tasks.Release.html#module-epmd-less-deployment).
```

Reply:
Could also mention https://www.erlang.org/news/140#improvements-and-new-features or the docs.

Reply:
I'd also suggest rephrasing the beginning too... but not sure how yet. the issue with `epmd-less mode` is it sounds both like you are referencing the project that is now no longer needed (https://github.com/tsloughter/epmdless) or that there is some `mode` while its a set of arguments to the VM startup, one of which is to not boot EPMD if its not running and another to set the port.

But *shrug* it may be fine and only confusing to me because I have other context in my head people won't.

Reply:
I agree with all points, except the `epmdless` project bit. I think that's only you @tsloughter :D

Reply:
Hah. Maybe. I've definitely seen it mentioned on Elixir slack at times by people who still think it is the way to get this done. There are a couple old blog posts still that will pop up when searching for this kind of stuff that lead the user down the old path of getting this done, in particular https://www.erlang-solutions.com/blog/erlang-and-elixir-distribution-without-epmd/

Reply:
That Erlang Solutions post being about `epmdless` specifically I mean.